packages/pwa/app/components/item-variant/item-price.jsx

```diff
@@ -24,6 +24,7 @@ const PricePerItem = ({currency, basket, basePrice}) => {
             />
             <FormattedMessage
                 defaultMessage="ea"
+                id="price_per_item.label.each"
```

Comment:
Nit: the location `price_per_item` is not consistent with the component name. I find `item-price` makes sense. Is there a reason why `price_per_item` is used instead?

Reply:
Because this one lives in a separate component called `PricePerItem`.

Reply:
oh I see, sorry i did not see the component there. It makes sense to me now 👍